src/main/java/randomizers/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package randomizers;
+
+import java.util.Random;
+import randomizers.sourcerand.Color;
+
+public class ColorSupplier {
+    private static final Color[] COLORS = Color.values();
+    private static final int SIZE = COLORS.length;
+    private static final Random RANDOM = new Random();
+
+    public Color getRandomColor() {
```

Comment:
according to Readme `getRandomColor()` must return `String`) so, you'll need not only to change the return type but also change the type of color in `Figure` and its subclasses to `String`